### PR TITLE
[14.0][ADD] stock_warehouse_analytic

### PR DESCRIPTION
**Pull request by rousseldenis**
Replaces #418

### CODE REVIEW

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers

File: stock_warehouse_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline stock_warehouse_analytic/__manifest__.py:1]: 2022 ?

File: stock_warehouse_analytic/models/stock_warehouse.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ACSONE SA/NV
```
Comment (xavier-bouquiaux) [inline stock_warehouse_analytic/models/stock_warehouse.py:1]: 2022 ?

**Review by xavier-bouquiaux — APPROVED** [review]
LGTM just a small change

File: stock_warehouse_analytic/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply stock_warehouse_analytic/__manifest__.py:1]: Because it was created at that time :-)

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers

**Review by sergiocorato — APPROVED** [review]

**Pull request merged**